pam/integration-tests/testdata/tapes/simple_auth.tape

```diff
@@ -0,0 +1,44 @@
+Output simple_auth.txt
+Output simple_auth.gif # If we don't specify a .gif output, it will create a default out.gif file.
+
+# Configuration header to standardize the output.
+# Does not work with the "Source" command.
+Set Width 1800
+Set Height 800
+Set FontFamily "Ubuntu Mono"
+Set FontSize 20
+Set Padding 0
+Set Margin 0
+Set Shell bash
+
+Hide
+Type "./pam_authd socket=/tmp/authd.sock"
```

Comment:
So we launch the process, but at the end of the tape, we never check for its return value...

So IMHO we should also do an `echo $?` once done to ensure that the process exited with the expected error.